packages/cli/test/unit/utils.unit.js

```diff
@@ -469,6 +469,17 @@ describe('Utils', () => {
     });
   });
 
+  describe('validateNotExisting', () => {
```

Comment:
> utils.validateNotExisting 

Seems strange to use a function with this name, when we are trying to figure out if '~' (root) is in the path...

Reply:
why not isRootInPath() ?

Reply:
it's the name of the function that it tests out